### PR TITLE
[FL2-6-FL2-455] Create ResourceBarChartArea component

### PR DESCRIPTION
**Pull request by st-build-service**
Fixed and raised by ricardo

### CODE REVIEW

File: components/bars/ResourceBarChartArea.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+export default class ResourceBarChartArea extends React.PureComponent {
+
+    static propTypes = {
+        className: PropTypes.string,
+        width: PropTypes.number,
+        height: PropTypes.number,
+        maxRelativeHeight: PropTypes.number,
+        bars: PropTypes.arrayOf(
+            PropTypes.arrayOf(
```
Comment (victorhtc) [inline components/bars/ResourceBarChartArea.js]: An array of arrays? Do you confirm it?

File: pages/componentsonly.js [hunk 2]
```diff
@@ -753,6 +755,159 @@ class BarDemo extends React.Component {
         );
     }
 }
+
+class StackedBarDemo extends React.Component {
```
Comment (victorhtc) [inline pages/componentsonly.js]: Use `React.PureComponent`

File: pages/componentsonly.js [hunk 3]
```diff
@@ -1382,7 +1537,9 @@ const theComponents = [
     {title:"Top Bar Container", percentComplete:90,comp : TopBarContainerDemo, width:"100%", type:"layout"},
 
     {title:"Nav Bar", percentComplete:90,comp : NavBarDemo, width:"80%", type:"layout"},
-    {title:"Bars", percentComplete:10,comp : BarDemo, type:"layout"},
+    {title:"Bars", percentComplete:100,comp : BarDemo, type:"layout"},
+    {title:"StackedBars", percentComplete:10,comp : StackedBarDemo, type:"layout"},
+    {title:"ResourceBarChartArea", percentComplete:10,comp : ResourceBarChartAreaDemo, type:"layout"},
```
Comment (victorhtc) [inline pages/componentsonly.js]: Format these lines.

File: components/bars/ResourceBarChartArea.js (hunk 1 above)
Comment (jimmed) [reply components/bars/ResourceBarChartArea.js]: This is definitely an array of arrays -- we are representing columns + rows here

Comment (jimmed) [reply components/bars/ResourceBarChartArea.js]: This is definitely an array of arrays -- we are representing columns + rows here.

**Pull request merged**